auth/src/services/authManager/express.ts

```diff
@@ -57,28 +58,64 @@ export const handleAuthIgnoreOnboarding = async (
   return UsersUseCases.getUserByOAuthUser(oauthUser)
 }
 
-export const handleApiSecretAuth = async (
+export const handleAdminAuth = async (
   req: Request,
   res: Response,
-): Promise<true | null> => {
+): Promise<boolean> => {
   const accessToken = req.headers.authorization?.split(' ')[1]
+  const provider = req.headers['x-auth-provider']
 
   if (!accessToken) {
     res.status(401).json({
       error: 'Missing or invalid access token',
     })
-    return null
+    return false
   }
 
   const isCorrectApiSecret = accessToken === config.apiSecret
-  if (!isCorrectApiSecret) {
+  if (isCorrectApiSecret) {
+    return true
+  }
```

Comment:
What is the point of `apiSecret` here?

Reply:
Backend instances retrieve a user information by public id using a permissioned endpoint, for handling access to these endpoints I use a api secret.